submissionaccesses.md

```diff
@@ -0,0 +1,48 @@
+# SubmissionAccesses Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/config/submissionaccesses**   
+
+Provide access to the existent configurations for the submission access. It returns the list of existent SubmissionAccessConfig.
+The SubmissionAccessConfig aggregates in a single place the configuration needed to the access section.
+
+## Single Submission-Form 
+**/api/config/submissionaccesses/<:access-name>**
```

Comment:
Minor change: If we rename the endpoint, this should be an `<:option-name>` instead of an `<:access-name>`.